media/css/firefox/channel.less

```diff
@@ -424,8 +424,12 @@ html[lang^="en"] {
 }
 
 #nightly header {
-    .at2x('/media/img/logos/firefox/nightly.png', 180px, 180px);
-    .background-size(180px 180px);
+    background-image: url('#{$image-path}/logos/firefox/browser/nightly/logo-2xl.png');
+    background-size: 180px 180px;
+
+    @media @highdpi {
+        background-image: url('#{$image-path}/logos/firefox/browser/nightly/logo-2xl-high-res.png');
```

Comment:
This logo isn't loading for me locally. I think that may be because `#{$image-path}` isn't a known / understood variable in Less?

Reply:
🤦‍♀ Not sure why it worked locally for me.